app/root.tsx

```diff
@@ -21,12 +23,14 @@ export function links() {
 
 export default function App() {
   return (
-    <html lang='en' className='h-full bg-teal-700'>
+    <html lang='en' className='h-full bg-gray-200'>
       <head>
         <Meta />
         <Links />
       </head>
-      <body className='h-full'>
+      <body className='h-full flex flex-col'>
+        <Nav />
+        <UserBar />
         <Outlet />
```

Comment:
Did you get it to work without the wrapper here?

Reply:
I did